modules/service/variables.tf

```diff
@@ -28,6 +28,16 @@ variable "service_discovery_namespace_id" {
   default = null
 }
 
+variable "deployment_circuit_breaker" {
+  type    = bool
+  default = true
+}
+
+variable "deployment_circuit_breaker_rollback" {
+  type    = bool
+  default = false
+}
+
```

Comment:
Could you provide some descriptions of what these variables do? Or possibly give them names to reflect the outcome they achieve?

e.g.

```
variable "stop_deployment_if_tasks_fail_to_start"
variable "rollback_deployment_if_tasks_fail_to_start"
```

or maybe this wants to be a three-way toggle:

```
variable "deployment_behaviour_if_tasks_fail_to_start"
```

with three options:

- "do nothing"
- "stop deployment"
- "roll back deployment"

Reply:
Agree comments are handy, have added some descriptions and comments linking out to AWS blog on the topic.